x-pack/plugin/core/src/main/java/org/elasticsearch/xpack/core/downsample/DownsampleAction.java

```diff
@@ -39,11 +43,18 @@ private DownsampleAction() {
     public static class Request extends MasterNodeRequest<Request> implements IndicesRequest, ToXContentObject {
         private String sourceIndex;
         private String targetIndex;
+        private TimeValue timeout;
```

Comment:
Maybe rename to wait timeout? To highlight that this is the timeout for waiting for the persistent task to complete?